### DISCUSSION TITLE
Batch Image Upload: Make columns of data sets sortable

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5901] [Feature Requests]
It would be awesome but not necessary for the initial release... being able to sort by column like in QB would be very helpful, especially for the progress status column!! 

I imagine just making the column headers clickable for users to sort the column they'd like -- ascending, descending, original order

<img width="1230" alt="Screenshot 2023-12-21 at 10 30 20 AM" src="https://github.com/specify/specify7/assets/135047322/a8f4910d-a548-4e78-882e-1d49bc6e864d">

<img width="1149" alt="Screenshot 2023-12-21 at 10 25 39 AM" src="https://github.com/specify/specify7/assets/135047322/7c99a05c-e6cd-4118-9fe0-9605632c7f40">

**Comment by realVinayak** [comment]
Fun fact: There was a time when you could do that. I removed it though.
